sources/packages/web/src/views/student/StudentAppealRequests.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <full-page-container>
```

Comment:
we have `<student-page-container`. u have used it in `sources/packages/web/src/views/student/StudentAssessmentAward.vue`

Reply:
Nice catch. fixed and pushed.